### ISSUE TITLE
problem installing

### ISSUE THREAD

**Issue by tarsiernyc** [statsmaths/cleanNLP#79]
sorry for the trouble - I am trying to install the program and get the following error message:

> cnlp_init_spacy()
Error: Python module 'cleannlp' not found. Install with:
  pip install cleannlp

when I try "pip install cleannlp" I get the following error:

> pip install cleannlp
Error: unexpected symbol in "pip install"

what am I doing wrong?

Thanks!

**Comment by taylor-arnold** [comment]
Have you installed Python on your machine, have a working version of pip?

**Comment by tarsiernyc** [comment]
Thanks for the quick reply. I have python on my machine and am able to import packages to python using pip.

**Comment by tarsiernyc** [comment]
Sorry for the trouble!

I just tried again- in r, im getting the following errors:

> cnlp_init_spacy()
Error: model en not found; use cnlp_download_spacy("en") to install
>

Then:
> cnlp_init_spacy("en")
Error: model en not found; use cnlp_download_spacy("en") to install

I then tried:

> cnlp_download_spacy("en_core_web_sm")
Error in py_call_impl(callable, dots$args, dots$keywords) : SystemExit: 1

Detailed traceback:
  File "C:\Users\psonk\ANACON~1\lib\site-packages\spacy\cli\download.py", line 50, in download
    sys.exit(dl)


Then I tried in the command prompt:

pip install -U cleannlp

I still get the same error:

> cnlp_init_spacy()
Error: model en not found; use cnlp_download_spacy("en") to install

I then tried to download spacy directly as per your suggestions:

https://spacy.io/usage.

But got the error: ModuleNotFoundError: No module named ‘spacy.pipleine.pipes’

Again, sorry for the trouble!

Thanks,

P
